test/integration/targets/nxos_smoke/tests/common/misc_tests.yaml

```diff
@@ -27,4 +25,3 @@
   nxos_command:
     commands:
       - show hostname
-    provider: "{{ connection | combine({'timeout': 500}) }}"
```

Comment:
This one concerns me a bit.  Pretty long timeout here which probably indicates there was a problem on some platforms..  We probably need to use the new method of setting the per task timeout using the `vars:` setting.  One of the 3 options I asked you to test for `nxos_gir`

Check the history for this file to see if the PR that added this indicates why it was added.

Reply:
This was not deliberate, I failed to notice this change in the sea of changes. Thanks for catching it.